run_tests

```diff
@@ -17,9 +17,9 @@ else
     safe_rg=cat
 fi
 
-export ONLY_RUN_FAST_TESTS=1
+export RUN_SLOW_TESTS_TOO=1
```

Comment:
this variable's name seemed to indicate it did the opposite of what it actually did, hence the switch.